crates/subspace-networking/src/create.rs

```diff
@@ -52,6 +52,12 @@ impl fmt::Debug for Config {
     }
 }
 
+impl Default for Config {
+    fn default() -> Self {
+        Config::new(identity::ed25519::Keypair::generate())
+    }
+}
```

Comment:
I added it at the last moment to make examples simpler. It is definitely expected for identity to be provided explicitly in production applications. You think I should remove it?

Reply:
Maybe it is better just to remove `Config::new` and make:
```rust
impl Config {
    pub fn with_key(keypair: Keypair) -> Self;
    pub fn with_generated_key() -> Self;
}
```

So that we could write it like this:
```rust
    let config_1 = Config {
        listen_on: vec!["/ip4/0.0.0.0/tcp/0".parse().unwrap()],
        value_getter: Arc::new(|key| {
            // Return the reversed key as a value
            Some(key.to_vec().into_iter().rev().collect())
        }),
        allow_non_globals_in_dht: true,
        ..Config::with_generated_key()
    };
```

Reply:
Good call, applied